### DISCUSSION TITLE
How to write a Readme.md file for an organization?

### DISCUSSION THREAD

**Discussion by Chicken2404** [community/community#133291] [Repositories] [answered]
### Select Topic Area

Question

### Body

How to write a Readme.md file for an organization? , i want to decorate my organization

**Comment by 0xLou1s** [comment, marked as answer]
check it bro 
https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/customizing-your-organizations-profile

**Comment by Chicken2404** [comment]
check it bro
https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/customizing-your-organizations-profile
